CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## [Unreleased]
+
+* Added:
+    * compatibility with OTP 23
```

Comment:
No kudos?

Reply:
I'm a selfless person, I've got no ego at all.

Fixed in ab0e350.